contracts/distributor.sol

```diff
@@ -32,7 +32,7 @@ contract Distributor {
     /// @dev Function that is called with an array of addresses for claiming tokens in their behalf.
     /// @param addresses Addresses of auction bidders that will be assigned tokens.
     function distribute(address[] addresses) public {
-        for (uint32 i = 0; i < addresses.length; i++) {
+        for (uint32 i = 0; i < addresses.length && auction.stage() != DutchAuction.Stages.TokensDistributed; i++) {
```

Comment:
There is already a check below `if (auction.bids(addresses[i]) > 0)` that cannot be true if the stage is `TokensDistributed`.
Do we really need this additional check?

Reply:
hmm no we don't